Terasology/ServerMOTD
# feat: Event-driven approach for message of the day

- Resolves #1 
- Implements a event-driven approach for displaying and editing MOTD, which works consistently as opposed to the previous implementation.

### How to test
- Start a game instance, use the `Host Game` option to run a game with the ServerMOTD module activated or start a headless server with the module added in the `override.cfg`.
- Start another game instance, use the Join Game option and select the game hosted in the previous step(it would be `localhost` under the Custom tab).
- Use the `displayMotd` command to see the 
- Run `overwriteMotd` and `appendToMotd` and test if the change is replicated on every client.

### To-do before merging
- [ ] MOTD should be displayed when a new client joins
- [ ] A message pop-up might be better than a chat message.
- [ ] Resolve bug with a client being able to run `overwriteMotd` command
